src/entity/Guild/Guild.ts

```diff
@@ -1,5 +1,6 @@
 import { Entity, PrimaryColumn, Column, OneToOne, OneToMany } from 'typeorm'
 import { GuildSettings, GuildSuggestion, GuildSupportTicket } from '.'
+import { GuildUser } from './GuildUser'
```

Comment:
Same here, once the class is added to the index.ts, you can change this import statement to
```
import { GuildUser } from '.'
```